### PR TITLE
[No Ticket] Clean up

### PR DESCRIPTION
**Pull request by Johnetordoff**
## Purpose

clean some issues on staging

## Changes

- add migration for license help text
- fix bug where the doi was stringified
- fix issue preventing email from being sent

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify
- Verify

What are the areas of risk?

Any concerns/considerations/questions that development raised?

### CODE REVIEW

File: osf/models/sanctions.py [hunk 1]
```diff
@@ -917,7 +917,7 @@ def _on_complete(self, event_data):
 
         self.save()
 
-        doi = registration.request_identifier('doi')
+        doi = registration.request_identifier('doi')['doi']
```
Comment (mfraezz) [inline osf/models/sanctions.py:920]: Should these be `.get('doi', None)`?

Comment (Johnetordoff) [reply osf/models/sanctions.py:920]: it should probably wake up Sentry if this fails, but just making the doi None makes it easy to identify times when datacite is down, though it probably fails prior to this.

Comment (mfraezz) [reply osf/models/sanctions.py:920]: Should it block execution of the rest of this function?

Comment (mfraezz) [reply osf/models/sanctions.py:920]: Looks like it's guaranteed to, or will break in a currently unhandled way anyway.

**Pull request merged**